lambdas/db-migration/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@cumulus/db-migration-lambda",
+  "description": "A Lambda function used for deploying DB migrations",
+  "version": "2.0.1",
+  "engines": {
+    "node": ">=12.18.0"
+  },
+  "private": true,
+  "scripts": {
+    "clean": "rm -rf dist",
+    "db:local:schema": "env PGPASSWORD=password pg_dump -h localhost -U postgres --schema-only --no-owner | egrep -v '^--' | cat -s | tee schema.sql",
```

Comment:
This is to print a copy of the schema locally? I'm trying to decide if this seems sufficient as a proxy to view the current schema of the database, since we won't have well defined models to do that.

Reply:
Yeah, that was what I was aiming for.

Reply:
I feel like we should consider a ticket to implement some sort of reference schema diagram generation from <the master database?>  via something like http://schemaspy.org/, that's been really helpful on other projects, but this sort of thing feels like it should work for <now>.

Reply:
I guess that's better than nothing - though I'm really discouraged that we're not going to have anywhere obvious in code defining the current state of our models.